LayoutTests/imported/w3c/web-platform-tests/css/css-writing-modes/forms/w3c-import.log

```diff
@@ -22,6 +22,14 @@ List of files:
 /LayoutTests/imported/w3c/web-platform-tests/css/css-writing-modes/forms/color-input-appearance-none-horizontal.optional.html
 /LayoutTests/imported/w3c/web-platform-tests/css/css-writing-modes/forms/color-input-appearance-none-vertical.optional-expected-mismatch.html
 /LayoutTests/imported/w3c/web-platform-tests/css/css-writing-modes/forms/color-input-appearance-none-vertical.optional.html
+/LayoutTests/imported/w3c/web-platform-tests/css/css-writing-modes/forms/meter-appearance-native-computed-style.optional.html
```

Comment:
Are we okay with doing a WPT import in the same commit as a code change, or should it be done separately?

Reply:
yeah, it's fine. This is only a partial import with the meter tests, there are no unrelated tests being imported, w3c-import.log reflects the current state of the tree.